jbang alias list . - fails with invalid json

I am building this project according to the instructions and have done the gradle install local maven repo, and then I am trying the jbang

```
jbang alias list .
[jbang] [WARN] Unable to read catalog /Users/davsclaus/workspace/tamboui/jbang-catalog.json because com.google.gson.JsonSyntaxException: com.google.gson.stream.MalformedJsonException: Unterminated object at line 68 column 6 path $.aliases..script-ref
See https://github.com/google/gson/blob/main/Troubleshooting.md#malformed-json
[jbang] [WARN] Unable to read catalog /Users/davsclaus/workspace/tamboui/jbang-catalog.json because com.google.gson.JsonSyntaxException: com.google.gson.stream.MalformedJsonException: Unterminated object at line 68 column 6 path $.aliases..script-ref
See https://github.com/google/gson/blob/main/Troubleshooting.md#malformed-json
[jbang] [ERROR] Unable to download catalog: /Users/davsclaus/workspace/tamboui/./jbang-catalog.json via null
[jbang] Run with --verbose for more details. The --verbose must be placed before the jbang command. I.e. jbang --verbose run [...]
~/workspace/tamboui main ❯
```

Also jq says it has a paring error
```
 cat jbang-catalog.json |jq                                                                                                                         ✘ INT
jq: parse error: Expected separator between values at line 68, column 21
~/workspace/tamboui main ❯
```

Yeah I can see a comma is missing at line 68
```
  63   │     "todo-list-demo": {
  64   │       "script-ref": "demos/todo-list-demo/src/main/java/dev/tamboui/demo/TodoListDemo.java"
  65   │     },
  66   │     "input-form-demo": {
  67   │       "script-ref": "demos/input-form-demo/src/main/java/dev/tamboui/demo/InputFormDemo.java"
  68   │     "colors-rgb-demo": {
  69   │       "script-ref": "demos/colors-rgb-demo/src/main/java/dev/tamboui/demo/ColorsRgbDemo.java"
  70   │     }
```

--- a/jbang-catalog.json
+++ b/jbang-catalog.json
@@ -65,6 +65,7 @@
     },
     "input-form-demo": {
       "script-ref": "demos/input-form-demo/src/main/java/dev/tamboui/demo/InputFormDemo.java"
+    },
     "colors-rgb-demo": {
       "script-ref": "demos/colors-rgb-demo/src/main/java/dev/tamboui/demo/ColorsRgbDemo.java"
     }

